core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-uve-device-selector/dot-uve-device-selector.component.html

```diff
@@ -33,10 +33,10 @@
         icon="pi pi-chevron-down"
         iconPos="right"
         [label]="'more' | dm"
-        [ngClass]="{ active: !$isADefaultDeviceActive() }"
+        [ngClass]="{ active: $isMoreButtonActive() }"
         styleClass="p-button-text p-button-sm"
         (click)="menu.toggle($event)"
         data-testId="more-button" />
 
-    <p-menu #menu [popup]="true" [model]="$menuItems()"></p-menu>
+    <p-menu #menu [popup]="true" [model]="$menuItems()" data-testId="more-options-btn" />
```

Comment:
This is not exactly the button, is the menu

Reply:
It's a test id

Reply:
Cool, it can be misleading when reading the test